build.xml

```
@@ -201,15 +207,34 @@
     <target name="saxon-unzip" depends="saxon-download" unless="${saxon-available}">
         <mkdir dir="${dir.lib.saxon}"/>
         <unzip src="temp/download" overwrite="true" dest="${dir.lib.saxon}"/>
+        <echo>downloaded saxon jar</echo>
+    </target>
+
+    <target name="schematron-download" unless="${schematron-available}">
+        <mkdir dir="${dir.lib.schematron}"/>
+        <get src="https://repo1.maven.org/maven2/com/helger/schematron/ph-schematron-ant-task/${schematron.version}/${schematron.jar.file}" dest="${dir.lib.schematron}"/>
+        <echo>downloaded schematron jar</echo>
     </target>
 
     <target name="xerces-download" unless="${xerces-available}">
         <mkdir dir="${dir.lib.xerces}"/>
         <get src="https://www.oxygenxml.com/maven/com/oxygenxml/oxygen-patched-xerces/${xerces.version}/${xerces.jar.file}" dest="${dir.lib.xerces}"/>
         <get src="https://www.oxygenxml.com/maven/com/oxygenxml/oxygen-basic-utilities/${xerces.version}/${oxygen.basic.utilites.jar.file}" dest="${dir.lib.xerces}"/>
+        <echo>downloaded xerces jars</echo>
+    </target>
+
+    <target name="init-mei-classpath" description="Initializes the mei.classpath, which is essential for the schema generation, by prepending the jar files contained in the lib directory to the java classpath. Will be called automatically if needed." depends="init">
+        <path id="mei.classpath">
+            <fileset dir="lib" erroronmissingdir="false">
+                <include name="saxon/*.jar"/>
+                <include name="xerces/*.jar"/>
+                <include name="schematron/*.jar"/>
+            </fileset>
```

I think we unintentionally delted:
```suggestion
            </fileset>
            <pathelement path="${java.class.path}"/>
```